src/main/java/com/acwong/peelplaylistcheck/model/ContentCollection.java

```diff
@@ -0,0 +1,40 @@
+package com.acwong.peelplaylistcheck.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class ContentCollection {
+    @JsonProperty("content")
+    private List<Content> contents = new ArrayList<Content>();
+
+    @JsonProperty("preroll")
+    private List<PreRoll> preRolls = new ArrayList<PreRoll>();
```

Comment:
Since currently Content and PreRoll are VideoGroups, what is the need for contents and preRolls to be lists?

Reply:
They are two different levels of lists.  Content/PreRoll derive from VideoGroup, which contains one or more Video's.  And at ContentCollection, it can contain one (or zero) or more of each of Content and PreRoll.  As such, they would need to be lists as well.  Please let me know if I miss something.